README.md

```diff
@@ -32,7 +32,7 @@ tar xf ./caddy_*.tar.gz
 sudo setcap cap_net_bind_service=+ep caddy
 ```
 
-Run `./caddy` with the following Caddyfile (replace the example values accordingly):
+Run `./caddy` with the following Caddyfile through the `-conf [Caddyfile]` option (replace the example values accordingly):
```

Comment:
Some opinions and suggests of mine: 
- Caddy v2 has a different argument of caddyfile (eg. `caddy adapt --config /path/to/Caddyfile`). So it will break in the future. Since Caddy is not a part of NaiveProxy, I think it probably ok without this explanation.
- Replace `[Caddyfile]` with `/path/to/Caddyfile`? `[Caddyfile]` looks like a option to me instead of a path.